sparkles/tests/test_checks.py

```diff
@@ -331,21 +347,34 @@ def test_guide_edge_check():
     acar = ACAReviewTable(aca)
     acar.check_catalog()
 
+    # For this constructed test we're interested in the edge marging warnings
```

Comment:
I would probably just fix the star positions so they don't overlap. I.e. `col=0` then `col=100`, `col=200` and same for `row` later.

Reply:
OK.  I think I started with that and then figured that if I changed the test more fundamentally, I'd need to justify any changes (and that it was still really doing the same thing) more than just acknowledging the new warnings?  But we can go that way.  Also a typo in my comment apparently.

Reply:
I'd say it is making the test better since it introduces variations in a parameter that should be independent.